serializers.py

```diff
@@ -70,7 +70,7 @@ class FolderSerializer(ModelSerializer):
     class Meta:
         model = Folder
         fields = '__all__'
-        read_only_fields = ['person', 'max_space', 'content_size']
+        read_only_fields = ['person', 'max_space', 'content_size','files']
```

Comment:
in line 64 files is already defined as read_only then why did you added it in read_only_fields

Reply:
Yes correct.
Made new commits.